add_sample_strategies.py

```
@@ -0,0 +1,191 @@
+            print(f"\n✅ Portfolio Summary Added:")
+            print(f"   • Total Strategies: {portfolio_summary['total_strategies']}")
+            print(f"   • Active Strategies: {portfolio_summary['active_strategies']}")
+            print(f"   • Total P&L: ${portfolio_summary['total_pnl_usd']:,.2f}")
+            print(f"   • Monthly Cost: {portfolio_summary['monthly_credit_cost']} credits")
+
+            # Test the API endpoint
+            print(f"\n🧪 Testing strategy portfolio retrieval...")
+            portfolio_result = await strategy_service.get_user_strategy_portfolio(user_id)
+
+            if portfolio_result.get("success", False):
+                print(f"✅ API Test Successful!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)